package.json

```diff
@@ -66,6 +66,9 @@
     ],
     "*.robot": [
       "python3 -m pipenv run robotidy --config tests/robot-tests/robotidy.toml"
+    ],
+    "*.py": [
+      "python3 -m pipenv run autopep8 --max-line-length 120 -r --in-place tests"
```

Comment:
Seems like we can use a `pyproject.toml` file to configure things like this instead. It'd also be good to have a script in `package.json` to run this too, so having the `pyproject.toml` would make the configurations consistent for these without needing to change the command in multiple places.